deployment/huggingface/yolos-seq2seq.ipynb

```diff
@@ -0,0 +1,1699 @@
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "KgGilFLFfgiw"
+      },
+      "source": [
+        "## 6. Model Register"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "gqTwdp7lhVlS",
+        "outputId": "b8cc3f44-0356-4020-ce94-585add15e8ef"
+      },
+      "outputs": [],
+      "source": [
+        "registered_model = client.get_or_create_registered_model(name='YOLOS - Hugging Face')"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "iDCNSz-zhjdj",
+        "outputId": "66babeb0-a96b-4a24-e245-690417a048e0"
+      },
+      "outputs": [],
+      "source": [
+        "model = registered_model.create_standard_model(\n",
+        "      name = 'v0',\n",
+        "      model_cls = Yolos,\n",
+        "      model_api = ModelAPI(yolos.example(), yolos.predict(yolos.example())),\n",
+        "      environment = Python(requirements = ['dill', 'Pillow', 'torch', 'transformers'])\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "RWuaPZE-fgf3"
+      },
+      "source": [
+        "## 7. Auto Scaling and Resources"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "tWqebcrEhWOb"
+      },
+      "outputs": [],
+      "source": [
+        "autoscaling = Autoscaling(min_replicas=1, max_replicas=20, min_scale=0.1, max_scale=10)\n",
+        "autoscaling.add_metric(CpuUtilizationTarget(0.6))\n",
+        "autoscaling.add_metric(MemoryUtilizationTarget(0.7))\n",
+        "autoscaling.add_metric(RequestsPerWorkerTarget(1))\n",
+        "resources = Resources(cpu=1., memory='4Gi')"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "XG597pl9fgZW"
+      },
+      "source": [
+        "## 8. Model Endpoint"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "-bqBsgLNizcs"
+      },
+      "outputs": [],
+      "source": [
+        "transformers_cache_env = {'TRANSFORMERS_CACHE': '/tmp'}"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "gjbfpxDYhWqX"
+      },
+      "outputs": [],
+      "source": [
+        "endpoint = client.get_or_create_endpoint('yolos-hugging-face')"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "4psmdLh4i3LR",
+        "outputId": "cf2c326a-4610-4b19-9a67-49e61c7418bb"
+      },
+      "outputs": [],
+      "source": [
+        "%%time\n",
```

Comment:
We should drop this magic command.  It's good while developing the notebook, but it draws attention to the fact that our endpoint updates can hang for a while.

Reply:
As discussed [here](https://vertaai.slack.com/archives/C048J9USF1B/p1672939554386629?thread_ts=1672863295.744129&cid=C048J9USF1B), I'll do this adjustment in a new PR :+1: